sys/dev/aac/aac_pci.c

```diff
@@ -493,6 +493,8 @@ static driver_t aacch_driver = {
 
 static devclass_t	aacch_devclass;
 DRIVER_MODULE(aacch, pci, aacch_driver, aacch_devclass, NULL, NULL);
+MODULE_PNP_INFO("U16:vendor;U16:device;U16:#;U16:#", pci, aacch,
```

Comment:
I don't think we need the trailing two U16:#; on these. You use those to skip bits that you need to.